templates/resources/alerting_rule.md.tmpl

```diff
@@ -0,0 +1,24 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "lightstep_alerting_rule Resource - terraform-provider-lightstep"
+subcategory: ""
+description: |-
+
+---
+
+# lightstep_alerting_rule (Resource)
+
+Provides a (Lightstep Alerting Rule)[https://api-docs.lightstep.com/reference/listalertingrules]. This can be used to create and manage Lightstep alerting rules.
```

Comment:
is the link syntax backward? ()[] rather than []()

Reply:
Good catch. This is borked in the current provider too. Whoops! https://registry.terraform.io/providers/lightstep/lightstep/latest/docs/resources/alerting_rule

I'll fix